_1327/minutes/views.py

```diff
@@ -104,7 +104,7 @@ def view(request, title):
 		'document': document,
 		'text': text,
 		'toc': md.toc,
-		'attachments': document.attachments.filter(downloadable=True).order_by('index'),
+		'attachments': document.attachments.filter(no_direct_download=True).order_by('index'),
```

Comment:
`no_direct_download` should be `False` because I've changed the semantics of that field